### ISSUE TITLE
sound not working 3ds

### ISSUE THREAD

**Issue by iliketoplayanddraw** [Mips96/Trogdor-Reburninated#9]
i tried the 3dsx and cia but i cant figure out how to get the audio working. im new to this stuff so i could just be being stupid lol

**Comment by Mode8fx** [comment]
Did you download directly from the repo or through Universal Updater? And do you have the music and sfx folders? They should be in `/3ds/Trogdor-RB` on your SD card.

**Comment by iliketoplayanddraw** [comment]
I tried both and have the folders but nothing will play in game

**Comment by Mode8fx** [comment]
I just redownloaded from the repo and it should work fine; are you sure you put the folders in the right place? Specifically, take the folder named `3ds` from the download and put it on the root of your SD card.

Though I did notice the download is currently broken in Universal Updater, so I just posted an issue to their repo to fix it. I think I changed the way the folder within the ZIP is named between versions so their app didn't know how to handle it.
